README.md

```diff
@@ -57,6 +57,10 @@ You may use [polyfills](https://www.npmjs.com/package/core-js) for `Set` and com
 
 If you want to get involved with Hull.js development, just use <a href="https://guides.github.com/introduction/flow/index.html" target="_blank">github flow</a> and feel free to contribute!
 
+## Typescript
```

Comment:
Hello, thanks for your pull request, but I don't maintain that .d.ts file and can't guarantee its correctness, that's why I would refrain from providing that link in the readme file.

Reply:
Hey @AndriiHeonia,
that's understandable. Then what about adding this file and export in hull.js repository to support Typescript and import syntax?

Reply:
Hosting d.ts files on DefinitelyTyped is a common practice, so I added link as you suggested initially. I just added some additional explanation. The change is already in the master branch, there is no need to do anything on your side, you can delete fork on your side if you want to avoid polluting your GitHub profile with forks. Thanks again for the pull request.

Reply:
sounds great. thanks!

Reply:
I gave it a 2nd thought eventually decided to consolidate everything (including NPM package) and d.ts file in one place: on GitHub. Also, I't like to let you know that during this exercise I did some pushes with `--force`, so your fork is diverged from the original repo, I'd recommend to re-fork it to avoid troubles with rebasing in the future (sorry for inconvenience).